src/resolvers.ts

```diff
@@ -1,8 +1,11 @@
 import { createUser } from './mutations/create-user';
 import { login } from './mutations/login';
+import { user } from './queries/user';
 
 export const resolvers = {
-  Query: {},
+  Query: {
+    user,
```

Comment:
Seguindo as sugestões do comment anterior, aqui vc faria:

```suggestion
    user: (_, args, context) => getUser(args, context),
```

Reply:
feito